src/Sleep.js

```diff
@@ -17,7 +17,13 @@ class Sleep {
 
   calculateDailySleep(id, date) {
     const findSleepByDate = this.sleepData.find((data) => id === data.userID && date === data.date);
-    return findSleepByDate.hoursSlept;
+    // this.sleepData.forEach(data => {
+    //   typeof data.hoursSlept === "number" ? data.hoursSlept : 0
+    // })
+    // typeof findSleepByDate.hoursSlept === "number" ? findSleepByDate.hoursSlept : 0
+    return this.sleepData.forEach(data => {
```

Comment:
What are we trying to return from this function? Returning forEach will always be 'undefined'.

Reply:
Has been reverted to old code.